tests/10apidoc/33room-members.pl

```diff
@@ -554,7 +523,7 @@ sub matrix_join_room_synced
       do => sub {
          matrix_join_room( $user, $room_id, %params );
       },
-      check => sub { exists $_[0]->{rooms}{join}{$room_id} },
+      check => sub { exists $_[0]->{rooms}{join}{$_[1]} },
```

Comment:
I'm *assuming* we're doing this so that we can join by alias. If so, could you s/room_id/room_id_or_alias/ elsewhere in this function?